truffleruby/src/main/java/org/truffleruby/core/CoreLibrary.java

```diff
@@ -789,6 +789,7 @@ private void initializeConstants() {
         Layouts.MODULE.getFields(rubiniusFFIModule).setConstant(context, node, "TYPE_VARARGS", RubiniusTypes.TYPE_VARARGS);
 
         Layouts.MODULE.getFields(objectClass).setConstant(context, node, "RUBY_VERSION", frozenUSASCIIString(RubyLanguage.RUBY_VERSION));
+        Layouts.MODULE.getFields(objectClass).setConstant(context, node, "TRUFFLERUBY_VERSION", frozenUSASCIIString(RubyLanguage.getTruffleRubyVersion()));
```

Comment:
Should it be `TRUFFLERUBY_VERSION` (like `JRUBY_VERSION`)
or `Truffle::VERSION` (like `Rubinius::VERSION`) ?